kion/resource_aws_account.go

```diff
@@ -298,77 +381,9 @@ func resourceAwsAccountCreate(ctx context.Context, d *schema.ResourceData, m int
 			PayerID:                   d.Get("payer_id").(int),
 		}
 
-		if v, exists := d.GetOk("aws_organizational_unit"); exists {
-			orgUnitSet := v.(*schema.Set)
-			for _, item := range orgUnitSet.List() {
-				if orgUnitMap, ok := item.(map[string]interface{}); ok {
-					postCacheData.OrganizationalUnit = &hc.PayerOrganizationalUnit{
-						Name:      orgUnitMap["name"].(string),
-						OrgUnitId: orgUnitMap["org_unit_id"].(string),
-					}
-				}
-			}
-		}
-
-		if rb, err := json.Marshal(postCacheData); err == nil {
-			tflog.Debug(ctx, "Creating new AWS account via POST /v3/account-cache/create?account-type=aws", map[string]interface{}{"postData": string(rb)})
-		}
```

Comment:
This was removed.  I found it helpful when debugging.  It's not entirely necessary that we leave it in, but it was helpful for me when debugging.

Reply:
Yeah, I removed it because I figured that folks could look at the Cloudwatch logs if an error would happen, but it makes sense to have it here for TF debugging.  I've added it back.